src/containers/Article/ArticleList.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react'
+import useFetechData from '@/src/hooks/useFetechData'
+import { ArticleListResponse } from '@/src/interfaces/Article.interface'
+import Article from './Article'
+import DropDown from '@/src/components/DropDown/DropDown'
+import SearchBar from '@/src/components/SearchBar/SearchBar'
+import Spiner from '@/src/components/Spinner/Spinner'
+import Button from '@/src/components/Button/Button'
+import styles from './Article.module.scss'
+
+export default function ArticleList() {
+  const [orderBy, setOrderBy] = useState('recent')
```

Comment:
```suggestion
  type OrderOption = 'recent' | 'like'
  const [orderBy, setOrderBy] = useState<OrderOption>('recent')
```
로 하면 조금더 타입세이프하게 가능할것같습니다! 
이렇게 바꾸면 여러부분에서 타입에러가 날것같은데 한번 고쳐보시면서 연습해보시면 좋을 것 같아요!

Reply:
혹은 이런방법도 있겠네요
```typescript
const orderOptions = {
  최신순: 'recent',
  인기순: 'like',
} as const

// ...
const [orderBy, setOrderBy] = useState<keyof typeof orderOptions>('최신순')
```